components/SiteNavigation.vue

```diff
@@ -118,7 +118,10 @@ const urlToActiveNav = {
   '/guides/farmlands-pos-integration': '/connections/farmlands/farmlands-pos-integration'
 };
 const route = useRoute();
-const currentPath = computed(() => urlToActiveNav[route.path] || route.path);
+function getCurrentPath() {
```

Comment:
This could still be a computed property. I think it's better as it avoids repeatedly calculating the same value.